src/main/java/com/box/sdk/BoxResource.java

```diff
@@ -1,5 +1,11 @@
 package com.box.sdk;
```

Comment:
it has to relate to your local environment, which version of checkstyle do you have?